Gemfile

```diff
@@ -69,7 +69,7 @@ group :test do
   gem 'rspec_junit_formatter'
   gem 'rspec-retry'
   gem 'semaphore_test_boosters'
-  gem 'selenium-webdriver', '3.142.7'
```

Comment:
I am not sure about this, so this is just a question, didnt we need this specific version for the selenium driver to work for showing us feature specs with the viewer?

Reply:
it's the same version in the gemfile.lock so I'm going to remove to keep inline with Hyku